solr/core/src/java/org/apache/solr/cloud/overseer/ZkWriteCommand.java

```diff
@@ -17,16 +17,37 @@
 package org.apache.solr.cloud.overseer;
 
 import org.apache.solr.common.cloud.DocCollection;
+import org.apache.solr.common.cloud.PerReplicaStates;
 
 public class ZkWriteCommand {
+
   public final String name;
   public final DocCollection collection;
+  /**
+   * if newState != null and priorState == null a new one is created
+   */
+
```

Comment:
What is the javadoc above documenting?

Reply:
There was a field there. this is a leftover. removed